app/views/profile/index.html.erb

```diff
@@ -1,17 +1,61 @@
-<h1>User Profile</h1>
-<table>
-  <thead>
-    <tr>
-      <th>User</th>
-    </tr>
-  </thead>
+<div class="jumbotron">
+  <h2 class="text-left"><%= current_user.email %></h2>
+  <p><a href="#" class="btn btn-info">Facebook</a>  <a href="#" class="btn btn-info">Twitter</a></p>
+  <p>Location: </p>
+  <p>About me: </p>
+  <td><%= link_to 'Edit' %></td>
```

Comment:
You don't need this line "<td><%= link_to 'Edit' %></td>"
We can worry about the edit profile functionality later.